addons/base_import/models/base_import.py

```diff
@@ -1462,6 +1534,53 @@ def execute_import(self, fields, columns, options, dryrun=False):
             import_skip_records=options.get('import_skip_records', []),
             _import_limit=import_limit)
         import_result = model.load(import_fields, merged_data)
+
+        # import translation if needed
+        if import_result['ids'] and translation_data:
+            model_terms_fields = [
+                fname for fname in translation_data if callable(model._fields[fname].translate)
+            ]
+            valid_langs = [lang for lang, _ in model.env['res.lang'].get_installed()]
+            if 'en_US' not in valid_langs:
+                valid_langs.append('en_US')
+            model.with_context(prefetch_langs=True).fetch(model_terms_fields)
+            records = model.browse(import_result['ids']).with_context(prefetch_langs=True)
+            for field_name, translation_data_ in translation_data.items():
+                field = model._fields[field_name]
+                if not field.translate:
+                    continue
+
+                for record, translation_data__ in zip(records, translation_data_):  # todo use strict=True
+                    # translation_data__: {'en_US': 'value1_en', 'fr_FR': 'value1_fr'}
+                    while field.related:
+                        record = record.mapped(field.related.rsplit('.', 1)[0])
+                        field = field.related_field
+
+                    assert field.store
+                    if field.translate is True:
+                        for lang, value in translation_data__.items():
+                            if lang in valid_langs:
+                                record.with_context(lang=lang, prefetch_langs=False)[field_name] = value
+                    else:
+                        # TODO support _lang
+                        value_en = record.with_context(lang='en_US')[field_name]
+                        if not value_en:
+                            continue
+                        value_neutralized = field.translate(lambda x: 'x', value_en)
+                        cache_value = {
+                            lang: value
+                            for lang in valid_langs
+                            # drop fallback values
```

Comment:
Here we actually drops all fallback values.
So, when I import language for fr_FR, for nl_NL,
its _nl_NL value will replace the nl_NL value.

I am OK, if we leave it as a non-ideal feature 😂 

Solving this is boring, you need to compare and pop to get the consice version of cache_value. That's why I recommend https://github.com/odoo/odoo/pull/224997/changes#r2627406115 which directly get the concise value from the database